### ISSUE TITLE
Feature Request: Choco installer to use same .MSI install as WinGet

### ISSUE THREAD

**Issue by gerardog** [gerardog/gsudo#212]
## Description
- Do not install to `C:\tools\gsudo` 
- Use `gsudoSetup.msi`

**Comment by ALERTua** [comment]
This would leave no options to install non-UAC gsudo #271

**Comment by soredake** [comment]
@ALERTua https://github.com/microsoft/winget-pkgs/pull/131849

**Comment by gerardog** [comment]
> This would leave no options to install non-UAC gsudo https://github.com/gerardog/gsudo/issues/271

@ALERTua how do you currently use chocolatey to make a non-UAC install?  I thougth you were using scoop for that, which works out of the box. 

@soredake: I appreciate your willing to help here, but creating a winget package is not just the creation of the manifest itself, but mainly updating the release pipeline to do that automatically on each release, and do the testing. 

Plus, having another installation method means a different 'gsudo experience' (for example, no gsudo powershell module, means worse experience plus updating all the docs), or that with user-scoped installs if you do `gsudo --user somebodyElse` you'll find out gsudo is not in the path of the target user so you cant gsudo anymore.

So let me ask you some questions.

- Does winget really offers a good portable experience such as scoop?
- Do you know if Winget Releaser (the tool I use to create winget packages) supports portable apps too?
    See https://github.com/gerardog/gsudo/blob/c23b1024b4974f5d90cf82968637bdb4c8924ca7/.github/workflows/release.yml#L140

- I see you are using net46 binaries. Can't x86, x64 and arm64 be used instead? Deprecating net46 was in the roadmap.

**Comment by soredake** [comment]
> Does winget really offers a good portable experience such as scoop?

No, not really, https://github.com/microsoft/winget-cli/issues/549 https://github.com/microsoft/winget-cli/issues/3077 https://github.com/microsoft/winget-cli/issues/222


> * Do you know if Winget Releaser (the tool I use to create winget packages) supports portable apps too?

https://github.com/russellbanks/Komac#update-an-existing-package-with-a-new-version

You can just use `komac update` after manifest is created.



> * I see you are using net46 binaries. Can't x86, x64 and arm64 be used instead? Deprecating net46 was in the roadmap.

Unfortunately, no, winget needs path to binary in order to create symlink (sudo -> net46-AnyCpu/gsudo) and you can't do per-arch symlinks and there is only all-in-one zip with all binaries.
